acssets/js/index.js

```diff
@@ -0,0 +1,48 @@
+const widthImg = 600;
+const slides = document.querySelector(".slides");
+const leftBtn = document.querySelector(".left-btn");
+const rightBtn = document.querySelector(".right-btn");
+const dotMain = document.querySelector(".dot-main");
+const dots = document.querySelectorAll(".dot");
+const img = document.querySelectorAll(".slide");
+slides.style.width = `${widthImg * img.length}px`;
+let sildeIndex = 0;
+dots[sildeIndex].className += " active";
+
+function makeSlideshow(n) {
+    if (n === 1) {
+        if (sildeIndex >= img.length - 1) {
+            sildeIndex = 0;
+        } else {
+            sildeIndex++;
```

Comment:
```suggestion
            sildeIndex + n;
```
viết như này thì e ko cần phải chia if else nữa, vì nếu n = -1 thì  slideIndex -- === slideIndex -1. E thử xem nhé

Reply:
vâng ạ